testsuite/shared/src/main/java/org/jboss/as/test/integration/security/common/servlets/SimpleServletWithScope.java

```diff
@@ -0,0 +1,79 @@
+package org.jboss.as.test.integration.security.common.servlets;
+
+import jakarta.annotation.security.DeclareRoles;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.HttpConstraint;
+import jakarta.servlet.annotation.ServletSecurity;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+import org.wildfly.security.http.oidc.OidcSecurityContext;
+
+
+/**
+ * Modified version of {@link SimpleServlet}. Checks token to find claims obtained using scope values in OIDC authentication
+ *
+ * @author Prarthona Paul
+ */
+@DeclareRoles({ SimpleServletWithScope.ALLOWED_ROLE })
+@ServletSecurity(@HttpConstraint(rolesAllowed = { SimpleServletWithScope.ALLOWED_ROLE }))
+@WebServlet(SimpleServletWithScope.SERVLET_PATH)
+public class SimpleServletWithScope extends SimpleServlet{
+    /** The serialVersionUID */
+    private static final long serialVersionUID = 1L;
+    public static final String SERVLET_PATH = "/SimpleServletWithScope";
+    public static final String ALLOWED_ROLE = "JBossAdmin";
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        OidcSecurityContext oidcSecurityContext = getOidcSecurityContext(req);
+
+        List<String> scopeClaims = new ArrayList<String>();
+        if (oidcSecurityContext != null) {
+            String scope = oidcSecurityContext.getToken().getClaimValueAsString("scope");
+            displayScopeClaims(scope, scopeClaims, oidcSecurityContext);
+        }
+
+        resp.setContentType("text/plain");
+        final PrintWriter writer = resp.getWriter();
+        if (Boolean.parseBoolean(req.getParameter(CREATE_SESSION_PARAM))) {
+            req.getSession();
+        }
+
+        writer.write(RESPONSE_BODY);
+        if(scopeClaims.isEmpty()) {
```

Comment:
Just a minor nit style wise - it's good to be consistent with white spaces. There should be a white space before the `(` here.